redux/thunks/userThunks.js

```diff
@@ -1,9 +1,17 @@
 import axios from 'axios';
-import {setUser, statusMessage, logInSuccess, loggedInFail} from '../actions';
+import {
+  setUser,
+  statusMessage,
+  getToken,
+  saveToken,
+  removeToken,
+  tokenError,
+} from '../actions';
 import {FAIL, SUCCESS} from './utils';
 import chalk from 'chalk';
 import {getCookie} from '../../utils';
 import {fetchClimbingRoutes} from './climbingRoutesThunks';
+import {AsyncStorage} from 'react-native';
```

Comment:
The _{AsyncStorage} from 'react-native';_ from react native might not work on some of the newer builds, the new and update way of importing will be : 

**import AsyncStorage from '@react-native-community/async-storage';**